common/Makefile.am

```diff
@@ -57,4 +57,9 @@ libva_display_la_CFLAGS	= $(libva_display_cflags)
 libva_display_la_LIBADD	= $(libva_display_libs)
 
 # Extra clean files so that maintainer-clean removes *everything*
+EXTRA_DIST =							\
+	loadsurface.h                                           \
+	loadsurface_yuv.h					\
+	$(NULL)
+
```

Comment:
loadsurface.h has been added on the release branch, I will remove this change. 

BTW on the release branch, loadsurface.h and loadsurface_yuv.h are added to EXTRA_DIST, not noinst_HEADERS.   do we need to change it ?